src/main/scala/nl.knaw.dans.easy.dd2d/mapping/package.scala

```diff
@@ -55,4 +56,17 @@ package object mapping {
 
     def toJsonObject: JsonObject = fields.toMap
   }
+
```

Comment:
We could later maybe move this to `dans-scala-lib`.

Reply:
The parameter should be renamed from `log` to something else; it could be used for other purposes than logging.